content/_data/site.yaml

```diff
@@ -11,11 +11,11 @@ styles:
   - screen.css
 facebook_id: 1820980378150914
 social:
-  email: birds
-  mastodon:
-    user: oddbird
-    server: front-end.social
-  github: oddbird
+  - email: birds
+  - mastodon:
+      user: oddbird
+      server: front-end.social
+  - github: oddbird
```

Comment:
For reasons that aren't entirely clear to me, the order of keys in this dict was inconsistent before. Making it an array now gives us control over the order.